db/model/model.js

```diff
@@ -103,3 +103,26 @@ exports.updateArticle = (article_id, votesInc) => {
         });
     });
 };
+exports.deleteCommentById = (comments_id) => {
+  return db
+    .query(`SELECT * FROM comments WHERE comment_id = $1`, [comments_id])
```

Comment:
You don't actually need to query the db here. If the comment doesn't exist then the `DELETE` will return no rows